packages/manager/apps/web-office/src/pages/dashboard/users/__test__/Users.spec.tsx

```diff
@@ -5,11 +5,12 @@ import { render } from '@/utils/test.provider';
 import usersTranslation from '@/public/translations/dashboard/users/Messages_fr_FR.json';
 import { licensesMock, usersMock } from '@/api/_mock_';
 
-describe('Users page', () => {
-  vi.mock('@/hooks', () => {
+describe('Users page for payAsYouGo', () => {
+  vi.mock('@/hooks', async () => {
```

Comment:
```suggestion
  vi.mock('@/hooks', () => {
```
We need also to do a test for prepaid licence if the behaviour isn't same.